CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Fixed issues with location handling and reworked some aspects of it's behaviour
 * Fixed potential issues by calculating Timestamp, dow, hours and tz from the same time moment
 * Fixed crashes that could happen on Application quit
+* CountlyConfiguration: EnableFirstAppLaunchSegment has been deprecated and this is going to be removed in the future.
```

Comment:
"EnableFirstAppLaunchSegment" in SDK configuration has been deprecated and it's functinality has been removed. This variable is going to be removed in the future.